### PR TITLE
travis: git-validation detects travis now

### PR DESCRIPTION
**Pull request by vbatts**
Signed-off-by: Vincent Batts <vbatts@hashbangbash.com>

### CODE REVIEW

**Comment by vbatts** [comment]
hmm. something strange happened in those failures...

**Comment by vbatts** [comment]
ah. found it. https://github.com/vbatts/git-validation/pull/6

**Comment by vbatts** [comment]
cool. PTAL.

**Comment by jonboulle** [comment]
but now why isn't this being tested?!?

**Comment by vbatts** [comment]
woah. it had been. though github was seeming to choke up a bit ago.

**Comment by vbatts** [comment]
https://travis-ci.org/opencontainers/runtime-spec/builds/120966194 ?
I think something is funky since the "specs" -> "runtime-spec" project renamed.

**Comment by vbatts** [comment]
@caniszczyk any thoughts? ^^

**Comment by caniszczyk** [comment]
I just kicked it again, looks good?

https://travis-ci.org/opencontainers/runtime-spec/builds/120966194

**Comment by vbatts** [comment]
@caniszczyk the build is fine, but not sure why travis status is no longer showing on PRs. I'm wondering if the project rename did not sync with travis?

**Comment by caniszczyk** [comment]
@vbatts ah you're right, wth! up for creating another PR to see if the issue fixed itself with a new PR?

I completely refreshed the Travis CI settings for the repos so not sure what's up yet.

**Comment by vbatts** [comment]
looks to be back. Thanks @caniszczyk

File: .travis.yml [hunk 1]
```diff
@@ -7,12 +7,12 @@ sudo: false
...
-  - go get github.com/vbatts/git-validation
+  - go get -u github.com/vbatts/git-validation
 
 install: true
 
 script:
   - go vet -x ./...
   - make .golint
-  - $HOME/gopath/bin/git-validation -run DCO,short-subject -v -range ${TRAVIS_COMMIT_RANGE}
+  - git-validation -run DCO,short-subject -v
```
Comment (wking) [inline .travis.yml]: Maybe want `dangling-whitespace` too (from vbatts/git-validation#7), although that somewhat of a drive-by with this PR.  Still, it would be nice to avoid things like #380.

**Comment by vbatts** [comment]
this PR should fix travis failures like this https://travis-ci.org/opencontainers/runtime-spec/jobs/122597352
when pushing a new tag

File: .travis.yml (hunk 1 above)
Comment (vbatts) [inline .travis.yml]: #381

**Comment by hqhq** [comment]
LGTM

**Comment by mrunalp** [comment]
Needs rebase.

**Comment by mrunalp** [comment]
LGTM
